internal/app/machined/pkg/controllers/k8s/manifest_test.go

```diff
@@ -292,6 +292,28 @@ func (suite *ManifestSuite) TestReconcileIPv6() {
 
 	v, _, _ = unstructured.NestedString(service.Object, "spec", "ipFamilyPolicy") //nolint:errcheck
 	suite.Assert().Equal("SingleStack", v)
+
+	r, err = suite.state.Get(
```

Comment:
`safe.StateGet`?

Reply:
copy-paste from the above, so decided to keep same style